### ISSUE TITLE
apacite issues in LaTeX preambles

### ISSUE THREAD

**Issue by fuhrmanator** [wwkong/Typesetting-Templates#1]
Hi -- the following lines in Notes_Template.lyx cause problems with the latest LyX:

% For Bibliography

\usepackage{apacite} % Order here is important!
\usepackage{natbib}

The latest LyX allows specifying natbib in the Bibliography prefs and, I can't really say what the trouble with apacite is (it's complicated). I just know that I commented out these lines from the LaTeX preamble created by your template and it works now.

**Comment by wwkong** [comment]
Thanks for the tip! I'll try some experimenting with the templates before I commit any changes.

**Comment by fuhrmanator** [comment]
Actually, I think the troubles go beyond apacite. I'm using babel (French documents) and a few other LyX things, and ran into troubles with LyX automatic changing (language substitution) of section names, formatting the headers wrong, etc. 

TL;DR - I started over from a blank document (no template) and just pasted the text from your Notes_Template.lyx template and tried to enable things properly via LyX 2.0 support (Document -> Settings). I was able to get everything to work with **nothing** in the LaTeX preamble. I'd be happy to send it back to you, or make a pull request. I didn't try any math stuff yet, however.
